### ISSUE TITLE
Possible reason for different performance between DIM Re-implementation and original implementation

### ISSUE THREAD

**Issue by VitoChien** [poppinace/indexnet_matting#29]
Thanks for your great work! 
When I'm trying to reproduce the result of DIM, I found that using **vgg model without BN** and **setting batch-size to 1** will give **the same or even better performance**, comparing to the results reported by the original DIM paper.

So I think that fewer training data is not supposed to be the reason for different performance.

**Comment by poppinace** [comment]
Solved. See [this response](https://github.com/poppinace/indexnet_matting/issues/17#issuecomment-818694167)
